assets/source/setup-guide/app/helpers/effects.js

```diff
@@ -54,3 +55,26 @@ export const useBodyClasses = ( style ) => {
 		};
 	}, [ style ] );
 };
+
+export const useConnectAdvertiser = () => {
+	return useCallback( async ( trackingAdvertiser, trackingTag ) => {
+		try {
+			const results = await apiFetch( {
+				path: `${ wcSettings.pinterest_for_woocommerce.apiRoute }/tagowner/`,
+				data: {
+					advrtsr_id: trackingAdvertiser,
+					tag_id: trackingTag,
+				},
+				method: 'POST',
+			} );
+
+			if ( trackingAdvertiser === results.connected ) {
+				return true;
+			} else {
+				return false;
+			}
```

Comment:
There is an ESLint error here. You can use ESLint plugins for your editor to catch this.

![image](https://user-images.githubusercontent.com/417342/148563133-2cc3c3e5-6ac2-45a6-baff-cc0be90d1490.png)